graphql/src/main/java/io/strimzi/admin/graphql/GraphQLService.java

```diff
@@ -0,0 +1,120 @@
+package io.strimzi.admin.graphql;
+
+import graphql.GraphQL;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.idl.SchemaGenerator;
+import graphql.schema.idl.SchemaParser;
+import graphql.schema.idl.TypeDefinitionRegistry;
```

Comment:
Could you stick this in your pom.xml:

```xml
<plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-dependency-plugin</artifactId>
                <version>${maven.dependency.version}</version>
                <executions>
                    <execution>
                        <id>analyze</id>
                        <goals>
                            <goal>analyze-only</goal>
                        </goals>
                        <configuration>
                            <failOnWarning>true</failOnWarning>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
```

You've got a direct dependency on graphql but you've not declared it in your pom, instead picking it up transitively via `vertx-web-graphql` I guess. Adding that to the pom will force you to declare all the stuff you depend on directly which makes for a more descriptive pom.

Reply:
Added to the parent pom.xml